tools/packaging/common/istio-iptables.sh

```diff
@@ -84,8 +83,8 @@ function isIPv4() {
 #
 # Function return true if agrument is a valid ipv6 address
 #
-ipv6section="^[0-9a-fA-F]{1,4}$"
 function isIPv6() {
+  ipv6section="^[0-9a-fA-F]{1,4}$"
```

Comment:
you need to use `local var="value"` in order to avoid leaking to global variable. Otherwise, var will still persist after function has been executed.